src/game/AI/ScriptDevAI/scripts/world/scourge_invasion.cpp

```diff
@@ -362,6 +374,52 @@ struct NecropolisHealthAI : public ScriptedAI
 {
     NecropolisHealthAI(Creature* creature) : ScriptedAI(creature)
     {
+        AddCustomAction(0, 5000u, [&]()
+        {
+            if (m_creature->GetHealthPercent() > 99.f)
+            {
+                CreatureList proxies;
+                GetCreatureListWithEntryInGrid(proxies, m_creature, NPC_NECROPOLIS_PROXY, 200.f);
```

Comment:
I need to get better values for the distance. This value works, but I'm sure it can be optimized.

Reply:
Can be reduced to 100.f